api/orchestrator/orchestrator.proto

```diff
@@ -136,13 +136,13 @@ service Orchestrator {
     };
   }
 
-  rpc ListRequirements(ListRequirementsRequest)
-      returns (ListRequirementsResponse) {
+  // List all metrics that are applicable to a certain control
+  rpc ListMetricsForControl(ListMetricsForControlRequest) returns (ListMetricsResponse) {
```

Comment:
Why do we need this? Isn't this returned by the database?

Reply:
When we load a catalog and its controls from the UI, we still need to load the metrics. So either we list all metrics and filter them in the UI, or we offer the filter directly in the API, what do you think?
Can we also add the option to specify a control in the ListMetrics rpc?

Reply:
"Can we also add the option to specify a control in the ListMetrics rpc?" Thats what I basically did just know ^^. If control id is set I call your implemented function (I just made it unexported)

Reply:
> When we load a catalog and its controls from the UI, we still need to load the metrics. So either we list all metrics and filter them in the UI, or we offer the filter directly in the API, what do you think?
> Can we also add the option to specify a control in the ListMetrics rpc?

Technically, this is not really needed from an UI point of view. In the UI we load all metrics once and store them in a global svelte store (as this information rarely changes), so when displaying a catalog we can filter the necessary metrics for the catalog coming out of the store, without triggering a new call to the backend.